vignettes/introduction-to-partition.Rmd

```diff
@@ -69,56 +71,90 @@ df <- simulate_block_data(
+
+Each variable in the original data set maps to variable in the reduced data set. In this partition, 63 reduced variables consist of two to seven variables each, as well as 1,076 of the original variables that did not get reduced because reducing them would lose too much information. Here are the top 20 clusters, ordered by how many raw features they represent:
+
+```{r}
+plot_ncluster(prt, show_n = 20) +
   # plot_*() functions return ggplots, so they can be extended using ggplot2
   theme_minimal(14)
 ```
 
-Each reduced variable explains at least 50% of the information of the original variables from which it was created. The distribution of information has a lower limit of our threshold, .5.
+Each reduced variable explains at least 50% of the information of the original variables that it summarizes. The distribution of information has a lower limit of our threshold, .5.
 
 ```{r}
 plot_information(prt, geom = geom_histogram) +
   theme_minimal(14)
 ```
 
-A key for these mappings and the information each variable explains can be retrieved with `mapping_key()`, which returns a nested `tibble`.
+Retrieve a key for these mappings and the information each variable explains with `mapping_key()`, which returns a nested `tibble`.
 
 ```{r}
 mapping_key(prt)
 ```
 
-To see each individual mapping, you can unnest them using `unnest_mappings()` (or do it yourself with `tidyr::unnest()`)
+To see each mapping, unnest them using `unnest_mappings()` (or do it yourself with `tidyr::unnest()`)
 
 ```{r}
 unnest_mappings(prt)
 ```
 
 ## Direct-Measure-Reduce and Partitioners
 
-Partitioners are functions that tell the partition algorithm 1) what to try to reduce 2) how to measure how much information is lost from the reduction and 3) how to reduce the data. We call this approach Direct-Measure-Reduce. In partition, functions that handle 1) are thus called directors, functions that handle 2) are called metrics, and functions that handle 3) are called reducers. partition has a number of pre-specified partitioners for agglomerative data reduction, but this approach is also quite flexible. See the [vignette on extending partition](extending-partition.html) to learn more about custom partitioners.
+Partitioners are functions that tell the partition algorithm 1) what to reduce 2) how to measure how much information and 3) how to reduce the data. We call this approach Direct-Measure-Reduce. In partition, functions that handle 1) are thus called directors, functions that handle 2) are called metrics, and functions that handle 3) are called reducers. partition has many pre-specified partitioners, but this approach is also quite flexible. See the [vignette on extending partition](extending-partition.html) to learn more about custom partitioners.
```

Comment:
Minor edits: 
In partition, functions that handle (1) are thus called directors, functions that handle (2) are called metrics, and functions that handle (3) are called reducers. partition has many pre-specified partitioners, but this aspect of the approach is also quite flexible.